CONTRIBUTING.md

```diff
@@ -37,6 +32,25 @@ it in the commit message (`Fixes #42.`).
 
 [mailmap]: https://github.com/git/git/blob/master/Documentation/mailmap.txt
 
+Setup
+-----
+
+1. Fork the repo.
+2. Install dependencies
+  Cram is used for tests:
```

Comment:
You may have to insert a new line before this one so that it doesn't end up on the same one as `Install dependencies`.

Currently:
![rcm 2015-01-05 11-46-25](https://cloud.githubusercontent.com/assets/602470/5616240/8f531d28-94d0-11e4-818f-a4596051bd9f.jpg)